src/AudioManager.js

```diff
@@ -19,7 +33,7 @@ export default class AudioManager {
         };
     }
 
-    playSound(soundName: string) {
+    playSound(soundName: CommandName) {
```

Comment:
In our current terminology, a "command" is the name for actions that can be done in a program. The commands currently available are: "forward", "left", and "right". So, the new `CommandName` type defined in `CommandPaletteCommand.js`, and used here, is broader than it should be. The set of available sounds is a super-set of the available commands: there are sounds for the commands plus sounds for some of the elements in the user interface. An alternative approach:

- Keep the new `CommandName` type but:
  - Move it to `types.js`, along side the `Program` type
  - Set it to `'forward' | 'left' | 'right'`
  - Update the `Program` type to `Array<CommandName>`
- Introduce a new type for the available sound names, such as `SoundNames`

I say "current terminology" above because the names that we have been using have been evolving through the project. We have a task for the next release to update the naming: https://issues.fluidproject.org/browse/C2LC-208

Reply:
Makes perfect sense, and done.